packages/components/src/spinner/style.scss

```diff
@@ -0,0 +1,44 @@
+.components-spinner--svg {
+	animation: 2s linear infinite both rotatesvg;
+	position: absolute;
+	top: 0;
+	left: 0;
+}
+
+@keyframes rotatesvg {
```

Comment:
Same comment with above, denoting the element as "track" might be better.

It seems correct to me to put this keyframe statement right in this components file where it will be used, especially since it's unique to this component and not reused. But naming-wise, we could use some BEM niceness here as well, for example `components-spinner__spin-track` perhaps?